br/tests/lightning_csv/run.sh

```diff
@@ -39,13 +41,24 @@ for BACKEND in tidb local; do
   check_contains 'id: 3'
   run_sql 'SELECT id FROM csv.empty_strings WHERE b <> ""'
   check_not_contains 'id:'
+}
 
-done
+rm -rf $TEST_DIR/lightning.log
+run_with "local" "tests/$TEST_NAME/config-pause-global.toml"
+grep -Fq 'pause pd scheduler of global scope' $TEST_DIR/lightning.log
```

Comment:
What's the meaning of these checks, should it assert "global" is found and "table" is not found?

Reply:
grep will fail if it cannot find it, not that explicit, copied from https://github.com/pingcap/tidb/blob/cde395dc597cab4540fe7aa141c5804ca8983519/br/tests/lightning_add_index/run.sh#L53

Reply:
shit, local run succeed

Reply:
```
              -e      Exit immediately if a pipeline (which may consist of a single simple command), a list, or a compound command (see SHELL GRAMMAR above), exits with a non-zero status.  The shell
                      does not exit if the command that fails is part of the command list immediately following a while or until keyword, part of the test following the if or elif reserved words, part
                      of any command executed in a && or || list except the command following the final && or ||, any command in a pipeline but the last, or if the command's return value is being
                      inverted with !.
```

> **The shell does not exit if ...... the command's return value is being inverted with !**

that means some integration test's check is not valid